### ISSUE TITLE
monoallelic_PAR

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7498]
**Preferred term label:**
monoallelic_PAR

**Synonyms**
PAR Dominant inheritance (requested)

**Definition (free text, please give PubMed ID)**
The allelic requirement for PAR Dominant inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP:0034340
